packages/coding-standard/config/symplify-rules.neon

```diff
@@ -205,12 +205,6 @@ services:
         class: Symplify\CodingStandard\Rules\MatchingTypeConstantRule
         tags: [phpstan.rules.rule]
 
-    -
-        class: Symplify\CodingStandard\Rules\ForceMethodCallArgumentConstantRule
-        tags: [phpstan.rules.rule]
-        arguments:
-            constantArgByMethodByType: []
-
```

Comment:
This can't be here, otherwise child configs will not allow argument override.
WTF